src/main/java/org/mule/module/apikit/error/ComponentExecutionExceptionHandler.java

```diff
@@ -6,20 +6,45 @@
  */
 package org.mule.module.apikit.error;
 
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
 import org.mule.runtime.api.component.execution.ComponentExecutionException;
 import org.mule.runtime.api.event.Event;
+import org.mule.runtime.core.api.event.CoreEvent;
+import org.mule.runtime.core.privileged.exception.MessagingExceptionUtils;
 
 /**
- * {@link RouterExceptionHandler} that propagates the resulting event in a {@link ComponentExecutionException}.
+ * {@link RouterExceptionHandler} that propagates the resulting event in a {@link
+ * ComponentExecutionException}.
```

Comment:
change this javadoc and the name of this class to not reference COmponentExecutionException

Reply:
also check the comment on DefaultFlowRoutingStrategy, it references this class